.circleci/config.yml

```diff
@@ -44,14 +44,19 @@ jobs:
     steps:
       - dev-setup
       - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
-      - run: INDEXER_DATABASE_URL="postgresql://postgres@localhost/postgres" RUST_BACKTRACE=full cargo nextest --nextest-profile ci --partition hash:1/1 --package smoke-test --retries 3
+      - run: echo "export RUST_BACKTRACE=full" >> $BASH_ENV
+      - run: echo "export INDEXER_DATABASE_URL=postgresql://postgres@localhost/postgres" >> $BASH_ENV
+      # --test-threads is intentionally set to reduce resource contention in ci jobs. Increasing this, increases job failures and retries
+      - run: cargo nextest --nextest-profile ci --partition hash:1/1 --package smoke-test --test-threads 6 --retries 3
```

Comment:
6 is still a lot, I remember we were at 3 before?

Reply:
5

Reply:
Why not 5 again? :)

Reply:
I like to live dangerously